src/App.js

```diff
@@ -449,6 +436,20 @@ class App extends Component {
                         <option value="Iota"> Iota (HD) </option>
                         <option value="Zeta"> Zeta (Series) </option>
                     </select>
+                    <select
+                        type="text"
+                        ref={this.searchInput}
+                        className="year-selector"
+                        id='mySelect'
+                        autoFocus={true}
+                        onChange={this.setYear}
+                        >
+                          <option value = 'default' className='op' >Default</option>
+                          <option value = '2020' className='op'  >2020</option>
+                          <option value = '2019' className='op' >2019</option>
+                          <option value = '2018' className='op' >2018</option>
+                        </select> 
```

Comment:
Enclose the **year filter select** and **movie engine select**  inside a `div`, as this will help in grouping them properly and improve the UI

Reply:
Done @silvareal

Reply:
Fix typo, **All Year** not **All Years**